tests/unit/test_mysqlsh_helpers.py

```diff
@@ -148,6 +149,112 @@ def test_configure_mysql_users_exception(self, _run_mysqlcli_script):
         with self.assertRaises(MySQLConfigureMySQLUsersError):
             self.mysql.configure_mysql_users()
 
+    @patch("mysqlsh_helpers.MySQL.get_cluster_status")
+    @patch("mysqlsh_helpers.MySQL._run_mysqlsh_script")
+    def test_remove_instances_not_online(self, _run_mysqlsh_script, _get_cluster_status):
+        """Test a successful execution of remove_instances_not_online."""
+        get_cluster_status_return = {
```

Comment:
might be worth storing this as a file and reading in later

Reply:
moving to a file may be overkill for now, but moved it to a constant at the top of file in a9bd5bb1a336943a9c0d601492f594aaddd073e8 (to be reused)